backend/addcorpus/migrations/0005_add_validators.py

```diff
@@ -4,6 +4,11 @@
 from django.db import migrations, models
 
 
+def validate_image_filename_extension(filename):
+    allowed = ['.jpeg', '.jpg', '.png', '.JPG']
```

Comment:
what about other filename extensions, for instance `webp`?

Reply:
Yes, this function is quite rudimentary, but it isn't used anymore, which is why it was moved to an older migration file. (See https://github.com/UUDigitalHumanitieslab/I-analyzer/pull/1526#discussion_r1557321862)